### DISCUSSION TITLE
How to change to: linking to category page instead of home

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1858] [Questions & Answers] [answered]
When I am at a specific category, I want the magento logo not to be linked to home, but linked to the category page. Eg. when I am at the category page no link, and when I have selected a product or some other page in this category, a link to the category page.

![home (2)](https://user-images.githubusercontent.com/13450219/136692511-125aeb97-ff7d-40ed-9531-804463435556.png)

**Comment by sreichel** [comment, marked as answer]
Seems its not possible. You have to rewrite `template/page/html/header.phtml` OR we add a method to the block class that can be used in layout updates (`<action method="setLogoUrl">` ).

**Reply by f1-outsourcing** [reply]
> Seems its not possible. You have to rewrite `template/page/html/header.phtml` OR we add a method to the block class that can be used in layout updates (`<action method="setLogoUrl">` ).


/template/page/html/header.phtml"

changing this

`$this->getUrl()`

to

`$curcat=Mage::registry('current_category'); $url = ($curcat)? '/'.$curcat->getUrlKey() : ''; echo $url `

However, I am not entirely sure this is an efficient solution. Is it not better getting this from the $this object?
